packages/web/pages/_app.tsx

```diff
@@ -14,13 +14,24 @@ import relativeTime from "dayjs/plugin/relativeTime";
 import utc from "dayjs/plugin/utc";
 import { GetKeplrProvider, useMatomoAnalytics } from "../hooks";
 import { IbcNotifier } from "../stores/ibc-notifier";
-import { IS_FRONTIER, NavBarEvents } from "../config";
+import {
+  AmplitudeEvent,
+  EventName,
+  IS_FRONTIER,
+  NavBarEvents,
+} from "../config";
+
+import * as amplitude from "@amplitude/analytics-browser";
```

Comment:
We could add this func to the hook, as we did with the matomo hook.

Then we could isolate this dependency.